pkg/cloudprovider/providers/azure/azure_storage.go

```diff
@@ -1,253 +0,0 @@
-/*
```

Comment:
AzureFile plugin does not implement provision-able volume interface. And does not manage storage accounts.  The only dependency this file had was azure disk plugin. And all the functionality has moved into the plugin itself. The new code is *single purpose* code optimized for the PD use case.